schemes/musica/musica_ccpp_util.F90

```diff
@@ -2,10 +2,15 @@
 ! SPDX-License-Identifier: Apache-2.0
 module musica_ccpp_util
 
+  use ccpp_kinds, only: kind_phys
+
   implicit none
 
   private
-  public :: has_error_occurred
+  public :: has_error_occurred, calculate_solar_zenith_angle_and_earth_sun_distance
+
+  real(kind_phys), parameter, public :: PI = 3.14159265358979323846_kind_phys
```

Comment:
I would suggest avoid hardcoding this in a separate location and use the CCPP standard name `pi_constant` if possible, or if not, maybe include from `shr_const_pi` (these are the same numerically). I realize this is used in another module and maybe the CCPP variable can be brought in from there.

Reply:
I guess my preference would be to use the `shr_const_pi` instead of adding yet another function argument to a list that is getting pretty long, and then having to pass constants down through all the internal functions, but I'll go with the approach the group prefers.